analysis.json

```diff
@@ -21,6 +21,40 @@
           }
         }
       }
+    },
+    "CVE-2022-1471": {
+      "dependency": {
+        "name": "org.yaml:snakeyaml",
+        "affectedVersions": [ "[,2.0)" ]
+      },
+      "assessments": {
+        "com.vaadin:vaadin-spring-boot-starter": {
+          "affectedVersions": {
+            "[14.0.0,23.3.5)": {
+              "status": "TRUE_POSITIVE",
+              "comment": "The snakeyaml dependency is a transitive dependency of the spring-boot-dependencies 2.x-3.1 dependency. Vaadin project is not depending on the vulnerable dependency (org.yaml:snakeyaml) directly, users can add the dependency if needed.",
```

Comment:
I would add this sentence at the end

You could add an exclusion to this dependency if your application does not use spring yaml configuration, or you are not using this library directly.

Reply:
or maybe mention that you can override the version by setting `snakeyaml.version` property to 2.0+

Reply:
updated